internal/pkg/scalable/scalableResource.go

```diff
@@ -36,10 +47,10 @@ type Workload interface {
 	GetObjectKind() schema.ObjectKind
 	// SetAnnotations sets the annotations on the resource. Changes won't be made on kubernetes until update() is called
 	SetAnnotations(annotations map[string]string)
```

Comment:
@jonathan-mayer I thought this method was inherited from a k8s api interface, isn't it? When I tried his twin method (GetAnnotations()) in the real cluster, it worked out of the box even without actual implementations within the codebase.  If I set it to private it will require me to do a custom implementations on every resource

Reply:
Oh your right, my bad.